src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/NormalizedMultiValuesTest.java

```diff
@@ -258,14 +270,44 @@ public void testGetAllNormalizedMultiValuesWfsJson20() throws Exception {
                         + "&outputFormat=application/json";
         JSONObject json = (JSONObject) getAsJSON(request);
         JSONArray features = json.getJSONArray("features");
-        assertEquals(2, features.size());
+        assertEquals(3, features.size());
         // check stations json content
         JSONObject station = getStationById(features, "st.1");
         assertNotNull(station);
         checkStationJson1(station);
         station = getStationById(features, "st.2");
         assertNotNull(station);
         checkStationJson2(station);
+        station = getStationById(features, "st.3");
+        assertNotNull(station);
+        checkStationJson3(station);
+    }
+
+    @Test
+    public void testGetAllNormalizedMultiValuesWfsJsonFormat20() throws Exception {
+        // check if this is an online test with a JDBC based data store
+        if (notJdbcBased()) {
+            // not a JDBC online test
+            return;
+        }
```

Comment:
Multi-values is only support with JDBC right?

Reply:
Right.